packages/genesys-spark-components/src/components/beta/gux-date-time/example.html

```diff
@@ -5,33 +5,40 @@ <h2>Date and Time</h2>
 <h3>Default</h3>
 <gux-date-time-beta></gux-date-time-beta>
 
-<h3>Short</h3>
-<gux-date-time-beta
-  datetime="2022-01-01T14:48:00.000Z"
-  format="short"
-></gux-date-time-beta>
-
-<h3>Medium</h3>
-<gux-date-time-beta
-  datetime="2022-01-01T14:48:00.000Z"
-  format="medium"
-></gux-date-time-beta>
+<div id="time-container"></div>
 
-<h3>Long</h3>
+<h3>Manually set time zone (America/Los_Angeles)</h3>
 <gux-date-time-beta
-  datetime="2022-01-01T14:48:00.000Z"
-  format="long"
+  time-zone="America/Los_Angeles"
+  format="full"
 ></gux-date-time-beta>
 
-<h3>Full</h3>
+<h3>Manually set datetime</h3>
 <gux-date-time-beta
   datetime="2022-01-01T14:48:00.000Z"
   format="full"
 ></gux-date-time-beta>
 
-<h3>Time zone</h3>
-<gux-date-time-beta
-  time-zone="America/Los_Angeles"
-  datetime="2022-01-01T12:00:00.000Z"
-  format="full"
-></gux-date-time-beta>
+<style
+  onload="(function () {
+  const timeContainer = document.getElementById('time-container');
+  [
+    'short', 'medium', 'long', 'full',
+  ].forEach((format) => {
+    const now = new Date();
+    const timestamp = now.toISOString()
+    const html = window.toHTML([
+      '<div>',
+        '<h3>' + format  + '</h3>',
+        '<gux-date-time-beta ' + 'format=' + format + ' >',
+        '</gux-date-time-beta>',
+      '</div>'
+    ].join(''));
+    timeContainer.appendChild(html);
+  });
+})()"
+>
+  #time-container h3 {
+    text-transform: capitalize;
+  }
+</style>
```

Comment:
is there a way to do this without making the whole component js generated? Like just doing the append to the components themselves?
Doing it this way feels like it takes away from the point of the docs site: seeing how the component is used in html in the code preview (and secondarily being able to compare it to the actual DOM). It makes it harder for consumers to understand how the component works.

I also am not quite understanding where the timestamp variable is used. It doesn't look like it's added to anything